azurerm/internal/services/postgres/postgresql_flexible_server_data_source.go

```diff
@@ -63,11 +63,6 @@ func dataSourcePostgresqlFlexibleServer() *pluginsdk.Resource {
 				Computed: true,
 			},
 
-			"cmk_enabled": {
```

Comment:
how come this is being removed? is it permanent or or a mistake in the swagger/sdk?

Reply:
In the past it was not possible to configure customer-managed keys (this feature does not exists for Azure Postgres Flexible Server). So I'm not expecting the return of `cmk_enabled` was really used.

However here is the feedback from the Product Group:
> This property was accidentally exposed in the preview version. Therefore, we removed it from the GA version. This will not be supported from backend going forward.

Reply:
ahh, instead of remove this which is a breaking change, could we instead deprecate it and just set it to am empty value instead? thanks!

Reply:
I have added deprecation notice for `cmk_enabled` and set empty value as suggested.